src/pybind/ceph_volume_client.py

```diff
@@ -1208,9 +1217,17 @@ def deauthorize(self, volume_path, auth_id):
         with self._auth_lock(auth_id):
             # Existing meta, or None, to be updated
             auth_meta = self._auth_metadata_get(auth_id)
+            if auth_meta is None:
```

Comment:
Probably, if auth-id doesn't exist and trying to de-authorize the same doesn't fall under idempotent operations where as the if the auth-id was authorized and the same is de-authorized multiple times, it can be considered idempotent operation. What do you think we should do here ? The same code is present in mgr/volumes as well.

Reply:
Will test_idempotency pass now? 
https://github.com/ceph/ceph/blob/cfb0c7e63ac63854cc2715e76ffd26563ca95364/qa/tasks/cephfs/test_volume_client.py#L315
I think it's OK that mgr/volumes has a different behavior. Let's not change the existing behavior of ceph_volume_client

Reply:
Done